packages/i18n-utils/src/locale-context.tsx

```diff
@@ -19,6 +20,22 @@ function getWpI18nLocaleSlug(): string | undefined {
 	return i18n.getLocaleData && i18n.getLocaleData()?.[ '' ]?.language;
 }
 
+/**
+ * Returns ISO 639 conforming locale string.
```

Comment:
I just keep naming the same as here. https://github.com/Automattic/wp-calypso/blob/175e89b3458751e78556fbebd1e37e473d8eaaca/apps/editing-toolkit/editing-toolkit-plugin/common/index.php#L190 But I agreed that calling `localeSlug` is better 👍 because I also saw the format is different from any types of ISO 639

Reply:
I renamed to `mapWpI18nLangToLocaleSlug` and use it in `getWpI18nLocaleSlug`